src/late_multicellular_stage/MulticellularHUD.tscn

```diff
@@ -37,6 +38,12 @@
 [ext_resource type="PackedScene" uid="uid://qwg2dluu8ow0" path="res://src/microbe_stage/PausePrompt.tscn" id="77"]
 [ext_resource type="PackedScene" path="res://src/gui_common/ActionButton.tscn" id="78"]
 
+[sub_resource type="ShaderMaterial" id="ShaderMaterial_32qwl"]
```

Comment:
I think it would be better to setup the damage effect as a separate scene so that all the stage HUDs can use it as an instance instead of needing all HUD scenes to have the shader etc. setup correctly.